engine/include/cubos/engine/renderer/viewport.hpp

```diff
@@ -0,0 +1,26 @@
+/// @file
+/// @brief Component @ref cubos::engine::Viewport.
+/// @ingroup renderer-plugin
+
+#pragma once
+#include <glm/glm.hpp>
```

Comment:
```suggestion
#pragma once

#include <glm/vec2.hpp>
```
Some space to breathe :relieved: also we try to use the specific include to have faster compile times